### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tgadiev**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -42,7 +35,7 @@ AWS java-6:
   tags:
     - aws
 
-AWS java-7:
+.AWS java-7:
```
Comment (vutkin) [inline .gitlab-ci.yml]: java 7 test should stay disabled?

File: defaults/main.yml [hunk 2]
```diff
@@ -1,33 +1,38 @@
...
 # - "local" Local artifact
+# - "repositories" Install openjdk from repositories for Debian and RHEL
 
-transport_web: "http://my-storage.com/jdk-8u172-linux-x64.tar.gz"
-transport_local: "/tmp/jdk-8u172-linux-x64.tar.gz"
+transport_web: '{{ transport_web_default }}'
+transport_local: "/tmp/openjdk-11.0.2_linux-x64_bin.tar.gz"
 
 transport_s3_bucket: s3_bucket
 transport_s3_path: /folder/jdk-8u172-linux-x64.tar.gz
```
Comment (vutkin) [inline defaults/main.yml]: Should default version for s3 transport be `8u172`?
transport_web_default point to 11 ver.

File: molecule/cloud-aws-direct-11/molecule.yml [hunk 3]
```diff
@@ -76,26 +79,32 @@ platforms:
 
 provisioner:
   name: ansible
-  log: False
+  log: True
```
[the remark this answers is not in the document]
Comment (tgadiev) [reply molecule/cloud-aws-direct-11/molecule.yml]: Fixed

File: .gitlab-ci.yml (hunk 1 above)
Comment (tgadiev) [reply .gitlab-ci.yml]: Yes, I suppose

File: tasks/fetch/security-fetch-s3.yml [hunk 4]
```diff
@@ -14,6 +14,6 @@
 
 - name: Downloaded security policy artifact
   set_fact:
-    security_policy_oracle_artifact: >
+    security_policy_java_artifact: >
       {{ download_path }}/{{  java_unlimited_policy_transport_s3_path
```
[the remark this answers is not in the document]
Comment (lab-lamz4) [reply tasks/fetch/security-fetch-s3.yml]: Fixed

File: tasks/fetch/web.yml [hunk 5]
```diff
@@ -10,19 +10,19 @@
```
[the remark this answers is not in the document]
Comment (lab-lamz4) [reply tasks/fetch/web.yml]: Done

File: tasks/fetch/security-fetch-web.yml [hunk 6]
```diff
@@ -10,4 +10,4 @@
 
 - name: Downloaded security policy artifact
   set_fact:
-    security_policy_oracle_artifact: '{{ file_downloaded.dest }}'
```
[the remark this answers is not in the document]
Comment (lab-lamz4) [reply tasks/fetch/security-fetch-web.yml]: Done

**Comment by ViachaslauKabak** [comment]
tasks\not-supported.yml
remove double quotes

**Pull request merged**